src/reporters/lang/en.js

```diff
@@ -25,6 +25,7 @@ const messages = {
   couldntClearPackageFromCache: "Couldn't clear package $0 from cache",
   clearedPackageFromCache: 'Cleared package $0 from cache',
   packWroteTarball: 'Wrote tarball to $0.',
+  operationAborted: 'Operation aborted.',
```

Comment:
This is not a user-friendly error message. Something more gentle and humanly like "Operation aborted upon user request." or "Installation canceled by the user." would be much better.